src/rollover.py

```diff
@@ -36,6 +37,36 @@ def _populate_ecs_info(self):
         info = self.ecs_client.describe_instances([self.ecs_id])
         self.ec2_id = info[self.ecs_id]['ec2InstanceId']
 
+        # store resource utilization
+        self.resources = {
+            "cpu": {
+                "registered": -1,
+                "remaining": -1,
+                "utilized": -1,
+            },
+            "memory": {
+                "registered": -1,
+                "remaining": -1,
+                "utilized": -1,
+            },
+        }
+
```

Comment:
Any chance this can just be `self.cpu_utilization` and `self.mem_utilization` ? Part of the idea with this class is to get away from all the nested dictionaries.